apps/web/src/pages/settings/SettingsPage.tsx

```diff
@@ -54,6 +56,11 @@ export function SettingsPage() {
 
   return (
     <SettingsPageWrapper>
+      <Test
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Remove before merge‏